lib/views/help/requesting.es.html.erb

```diff
@@ -477,22 +474,23 @@
         <strong>El Habeas Data</strong> garantiza la protección de los datos privados y el acceso de toda persona a cualquier información que pueda tener cualquier entidad pública sobre ella, así como el derecho a saber por qué y con qué finalidad tiene esa información. ( Ver Punto 6)
       </p>
       <h2>
-        Los recursos en Nicaragua
+        Los recursos en Nicaragua <a href="#recursos">#</a>
       </h2>
       <p>
-        En caso que la entidad niegue el acceso a la información, entregue información incompleta o no conteste dentro del término establecido por la LAIP, la persona solicitante podrá:
+        En caso que la entidad niegue el acceso a la información, entregue información incompleta o no conteste dentro del término establecido por la LAIP, la persona solicitante podrá (según la Ley de Acceso a la Información Pública y su Reglamento):
       </p>
       <ol>
-        <li>Apelar por escrito o por vía electrónica ante la Coordinación de Acceso a la Información Pública que corresponda, dentro del término de 6 días de notificada la resolución. Esta instancia deberá resolver la apelación en un término de 30 días, agotándose con ella la vía administrativa. Si el escrito de apelación está incompleto, la Coordinación de Acceso a la Información Pública correspondiente deberá completar dichas omisiones.
+        <li>1)	Apelar por escrito o por vía electrónica ante la Coordinación de Acceso a la Información Pública que corresponda, dentro del término de 6 días de notificada la resolución. Esta instancia deberá resolver la apelación en un término de 30 días, agotándose con ella la vía administrativa. Si el escrito de apelación está incompleto, la Coordinación de Acceso a la Información Pública correspondiente deberá completar dichas omisiones. <a href="http://legislacion.asamblea.gob.ni/Normaweb.nsf/($All)/675A94FF2EBFEE9106257331007476F2">(Ver artículos 37 y 38 de la Ley de Acceso a la Información Pública)</a>
```

Comment:
Don't need the `1)` as that's handled by the `<ol>` styling:

![screen shot 2015-03-16 at 12 34 06](https://cloud.githubusercontent.com/assets/282788/6666091/c20d7e68-cbd8-11e4-82df-21998cff2e29.png)

Will take care of this in merge